frontend/src/components/Schedules/EditScheduleForm.tsx

```diff
@@ -58,61 +63,69 @@ export const EditScheduleForm: FC<Props> = ({ onUpdate, schedule }) => {
       <div className={flexWrapping}>
         <InlineFormLabel tooltip={intl.get('group_name')}>{intl.get('name')}</InlineFormLabel>
 
-        <Input
-          onChange={({ currentTarget: { value } }) => onUpdate(ScheduleKey.NAME, value)}
-          name={intl.get('name')}
-          defaultValue={schedule.name}
-          placeholder={intl.get('name')}
-          css=""
-        />
+        <div className={flexContainer}>
+          <Input
+            onChange={({ currentTarget: { value } }) => onUpdate(ScheduleKey.NAME, value)}
+            name={intl.get('name')}
+            defaultValue={schedule.name}
+            placeholder={intl.get('name')}
+            css=""
+          />
+        </div>
       </div>
 
       <div className={flexWrapping}>
         <InlineFormLabel tooltip={intl.get('group_description')}>{intl.get('description')}</InlineFormLabel>
 
-        <Input
-          onChange={({ currentTarget: { value } }) => onUpdate(ScheduleKey.DESCRIPTION, value)}
-          name={intl.get('description')}
-          defaultValue={schedule.description}
-          placeholder={intl.get('description')}
-          css=""
-        />
+        <div className={flexContainer}>
+          <Input
+            onChange={({ currentTarget: { value } }) => onUpdate(ScheduleKey.DESCRIPTION, value)}
+            name={intl.get('description')}
+            defaultValue={schedule.description}
+            placeholder={intl.get('description')}
+            css=""
+          />
+        </div>
       </div>
 
       <div className={flexWrapping}>
         <InlineFormLabel tooltip={intl.get('report_interval_description')}>
           {intl.get('report_interval')}
         </InlineFormLabel>
 
-        <Select
-          value={getIntervals().filter((interval: any) => interval.value === schedule?.interval)}
-          options={getIntervals()}
-          onChange={(selected: SelectableValue) => {
-            onUpdate(ScheduleKey.INTERVAL, selected.value);
-          }}
-        />
+        <div className={flexContainer}>
+          <Select
+            value={getIntervals().filter((interval: any) => interval.value === schedule?.interval)}
```

Comment:
Might wanna define an interface `ITimeInterval` or something? e.g.

```
export interface ITimeInterval {
  label: string;
  value: number;
}
```

Aside: may also want to define some constants for the interval values (enums or POJOs). If add enums, can make the interface even more typesafe (but less flexible):

```
export interface ITimeInterval {
  label: string;
  value: ETimeInterval;
}
```